models/neurons/iaf_psc_delta_fixed_timestep_neuron.nestml

```diff
@@ -0,0 +1,98 @@
+"""
+iaf_psc_delta_fixed_timestep - Current-based leaky integrate-and-fire neuron model with delta-kernel post-synaptic currents
+###########################################################################################################################
+
+Description
++++++++++++
+
+iaf_psc_delta is an implementation of a leaky integrate-and-fire model
+where the potential jumps on each spike arrival.
+
+The threshold crossing is followed by an absolute refractory period
+during which the membrane potential is clamped to the resting potential.
+
+Spikes arriving while the neuron is refractory, are discarded by
+default. If the property ``with_refr_input`` is set to true, such
+spikes are added to the membrane potential at the end of the
+refractory period, dampened according to the interval between
+arrival and end of refractoriness.
+
+The general framework for the consistent formulation of systems with
+neuron like dynamics interacting by point events is described in
+[1]_.  A flow chart can be found in [2]_.
+
+This model differs from ``iaf_psc_delta_fixed_timestep`` in that it
```

Comment:
This is certainly a typo and should read:
`This model differs from ``iaf_psc_delta`` ...`